lib/streamlit/report_session.py

```diff
@@ -405,6 +407,13 @@ def _enqueue_new_report_message(self):
         msg.new_report.id = self._report.report_id
         msg.new_report.name = self._report.name
         msg.new_report.script_path = self._report.script_path
+        deploy_params = self._repo.get_repo_info()
+        msg.new_report.is_deployable = deploy_params is not None
+        if deploy_params is not None:
```

Comment:
What's your opinion (here and elsewhere) of just dropping "is not None", using the python in built check for truthiness instead?

Reply:
so I can do `if not deploy_params:`? I'll check. I'll see what our convention is.

Reply:
Based on Google Style guide and PEP-8 `is not None` is a normal and preferred option. We also use it everywhere.

[https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#2144-decision](https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#2144-decision)

[https://www.python.org/dev/peps/pep-0008/#programming-recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)